neuromation/cli/job.py

```diff
@@ -176,8 +191,8 @@ def job() -> None:
     multiple=True,
     help="Mounts directory from vault into container. "
     "Use multiple options to mount more than one volume. "
-    "--volume=HOME is an alias for storage://~:/var/storage/home:rw and "
-    "storage://neuromation/public:/var/storage/neuromation:ro",
+    f"--volume=HOME is an alias for storage://~:/var/storage/home:rw and "
```

Comment:
Very minor: `f` is redundant here and below.

Reply:
oops :upside_down_face: